### PR TITLE
<h1> Header added to E-Commerce App 

### PR DESCRIPTION
**Pull request by sara-no-h**
added <h1> Header to index.html for E-Commerce App

### CODE REVIEW

**Comment by bb-codes6** [comment]
Not sure about h1 code.

**Review by bb-codes6 — COMMENTED** [review]
h1 changes

**Comment by sara-no-h** [comment]
What do you need me to fix with the h1 code?

**Comment by sara-no-h** [comment]
I am also not seeing the resolve conversation button as described in the project outline.

File: index.html
Comment (sara-no-h) [inline index.html:1]: What would you like me to change on the h1?

**Review by bb-codes6 — CHANGES_REQUESTED** [review]
Add color and remove text

**Review by bb-codes6 — APPROVED** [review]

**Pull request merged**